### ISSUE TITLE
Spoken message appears as a short gap of silence if streaming/playing

### ISSUE THREAD

**Issue by svrooij** [svrooij/node-sonos-ts#38]
Issue with sonos2mqtt see https://github.com/svrooij/sonos2mqtt/issues/73

**Comment by svrooij** [comment]
1. create a fork of [node-sonos-ts](https://github.com/svrooij/node-sonos-ts).
 2. run `npm install` in the repository
 3. Change the code you want
 4. Press F5 (in VS Code), while the [play-notification sample](https://github.com/svrooij/node-sonos-ts/blob/master/examples/play-notification.js) is open.

That should build (actually transpile) the typescript to javascript, and run the current open sample. The code in the repository is only TypeScript and the **lib**-folder is automatically ignored. 

> That won't help because the AsyncHelper isn't used when setting the Volume. Maybe it should be used. However that's above my experience level.

I mean that it might be better to put the delay function in the [AsyncHelper](https://github.com/svrooij/node-sonos-ts/blob/master/src/helpers/async-helper.ts) class. That code might be needed in other places in the library. Something like

```typescript
static async Delay(milliseconds: number): Promise {
  return await new Promise(resolve => setTimeout(resolve, 500));
}
```

Then you can use it by calling:
```typescript
await AsyncHelper.Delay(150);
```

**Comment by svrooij** [comment]
@besynnerlig if you have fixed this issue be sure to put the following in the commit message, so it will automatically release after merge.

```plain
fix: TTS not playing

Fixed #38
```

The `fix: ` will signal a new patch version. The `Fixed #38` will close this issue after merging.
